### ISSUE TITLE
Feed Me error: unknown property: venveo\redirect\elements\Redirect::sectionId

### ISSUE THREAD

**Issue by daltonrooney** [venveo/craft-redirect#15]
I'm trying to run a CSV import via FeedMe and am getting the following error when running the feed:

```
{"date":"2019-12-19 12:24:00","type":"info","message":"Redirects: Preparing for feed processing."}
{"date":"2019-12-19 12:24:00","type":"info","message":"Redirects: Finished preparing for feed processing."}
{"date":"2019-12-19 12:24:00","type":"info","message":"Redirects: Starting processing of node `#1`.","key":"KfsTDQUnVPmUw937cDzw"}
{"date":"2019-12-19 12:24:00","type":"info","message":"Redirects: Match existing element with data `{\"sourceUrl\":\"\\\/about\\\/contact\\\/\"}`.","key":"KfsTDQUnVPmUw937cDzw"}
{"date":"2019-12-19 12:24:00","type":"error","message":"Redirects: `Getting unknown property: venveo\\redirect\\elements\\Redirect::sectionId - Component.php: 154`.","key":"KfsTDQUnVPmUw937cDzw"}
{"date":"2019-12-19 12:24:00","type":"info","message":"Redirects: Processing 0 elements finished in 0.02s"}
```

Craft CMS 3.3.18.2
FeedMe 4.1.2
Craft Redirect 2.0.12.1

Any thoughts?

**Comment by Mosnar** [comment]
Oh hey, I didn't realize anyone was using this unpublished plugin! Looks like a small bug - should be able to knock it out while I do a 3.4 audit

**Comment by daltonrooney** [comment]
Oh that’s funny - I didn’t realize this was unpublished! I think I may have found it via Discord and installed via Composer. We’ve used it on a couple of sites now and really like it, so thanks for maintaining it!

**Comment by Mosnar** [comment]
Should be fixed in 2.1.0 which is just released! I made quite a few changes and improvements in preparation for the major changes I intend to make once Craft 3.4 comes out, so make sure you backup things properly and make sure things still work as expected and let me know if you have any issues.

**Comment by Mosnar** [comment]
I noticed 2.1.0 failed to automatically publish to packagist. That's fixed.
